.github/workflows/release.yml

```
@@ -0,0 +1,31 @@
+name: release
+on:
+  release:
+    types: [ created ]
+
+permissions:
+  contents: write
+
+jobs:
+  create-release:
+    name: Release Go Binary
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goos: [ linux, darwin ]
```

Seems like `wanyoucao577/go-release-action` supports windows builds. Can you maybe try adding 'windows' to the build matrix? 

```suggestion
        goos: [ linux, darwin, windows ]
```

(You might also need to exclude windows+arm64 from the matrix)

Even if the windows binaries don't work out-of-the box for everyone, it would probably be better than nothing. If someone has issues running the binary they can open an issue and we can investigate further.